mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/180 tab look and feel

Closes #180 

To test
 - run the [Test] Main UI scene
 - navigate to the Zone or Component View
 - Observe the tabs across the top of the view to verify they look like the concept art
